mod/notifications/start.php

```diff
@@ -116,6 +118,40 @@ function _notifications_page_menu($hook, $type, $return, $params) {
 	return $return;
 }
 
+/**
+ * Register menu items for the title menu on group profiles
+ *
+ * @param \Elgg\Hook $hook 'register' 'menu:title'
+ *
+ * @return void
+ *
+ * @access private
+ * @since 3.0
+ */
+function _notification_groups_title_menu(\Elgg\Hook $hook) {
+	if (!elgg_is_active_plugin('groups')) {
+		return;
+	}
+
+	$user = elgg_get_logged_in_user_entity();
+	if (!$user) {
+		return;
+	}
+
+	$items = $hook->getValue();
+	/* @var ElggMenuItem[] $items */
+
+	foreach ($items as $item) {
```

Comment:
why looping this? why not add directly as a child?

Reply:
I don't see any other way other than adding in the groups plugin, but I wanted to keep this in notifications.

Reply:
If there's a better way let me know.